src/main/java/sirius/biz/web/BizController.java

```diff
@@ -124,6 +125,17 @@ private HandledException invalidTenantException() {
         return Exceptions.createHandled().withNLSKey("BizController.invalidTenant").handle();
     }
 
+
+    /**
+     * Checks whether the current user is an account of the system tenant. This can be useful for granting extra features.
+     *
+     * @return <tt>true</tt> if the user is an account of the system tenant, <tt>false</tt> otherwise.
+     */
+    protected boolean isUserAccountOfSystemTenant() {
+        TenantUserManager<?, ?, ?> userManager = tenants.getTenantUserManager();
+        return userManager.getSystemTenantId().equals(userManager.getOriginalTenantId());
```

Comment:
```suggestion
        return Strings.areEqual(userManager.getSystemTenantId(), userManager.getOriginalTenantId()));
```


...you just never know...